packages/forma-36-website/src/components/ComponentSource.js

```diff
@@ -38,7 +40,7 @@ class ComponentSource extends React.Component {
       <div className="f36-margin-bottom--m">
         <LiveProvider
           code={this.props.children.trim()}
-          scope={{ ...f36Components, tokens }}
+          scope={{ ...f36Components, tokens, ...f36icons, Icon }}
```

Comment:
I think we need to do `scope={{  tokens, ...f36icons, ...f36Components, Icon }}`, `Heading` is a Forma36 component and not an icon